httpx/ssrf.go

```diff
@@ -20,47 +21,6 @@ type noInternalIPRoundTripper struct {
 	internalIPExceptions        []string
 }
 
-// NewNoInternalIPRoundTripper creates a RoundTripper that disallows
-// non-publicly routable IP addresses, except for URLs matching the given
-// exception globs.
-func NewNoInternalIPRoundTripper(exceptions []string) http.RoundTripper {
```

Comment:
Don't remove this if you want to keep BC :)

Reply:
Yeah, good point.